src/components/member/tabs/contracts-tab/contract/master-inception.tsx

```diff
@@ -54,13 +56,12 @@ export const MasterInception: React.FC<{
               fullWidth
               disabled={activateContractLoading}
               onClick={() => {
-                const confirmed = window.confirm(
-                  `Are you sure you want to activate this contract with master inception of ${format(
-                    activeFrom,
-                    'yyyy-MM-dd',
-                  )}?`,
-                )
-                if (confirmed) {
+                const confirmMsg = `Are you sure you want to activate this contract with master inception of ${format(
```

Comment:
Preferably no unnecessary abbreviations
```suggestion
                const confirmMessage = `Are you sure you want to activate this contract with master inception of ${format(
```

Reply:
Position "center top" is relevant only for Confirm-like modals. For other modals I think usual position is center